lib/utils/src/buildinfo.ts

```diff
@@ -101,10 +102,24 @@ function readBuildInfo() {
 
   // None of the candidate file paths led to success. Fall back to env.
   if (!process.env.OPSTRACE_BUILDINFO_PATH) {
-    throw Error(
-      `environment variable OPSTRACE_BUILDINFO_PATH not set ` +
-        `and none of [${JSON.stringify(CANDIDATE_PATHS)}] found/readable`
-    );
+    if (process.env.NODE_ENV === "development") {
+      log.info(
+        "using hardcoded 'local-dev' for BUILD_INFO, to override set environment variable OPSTRACE_BUILDINFO_PATH"
```

Comment:
>  override set environment variable OPSTRACE_BUILDINFO_PATH

Hm. I don't think this is logically correct.

It's more of a "seen NODE_ENV === "development", so do not require OPSTRACE_BUILDINFO_PATH to be set"

More importantly, in my experimentation with this approach I deliberately did _not_ import `log` because I ran into a circular import -- buildinfo.ts as well as log.ts are candidates for being imported super early.